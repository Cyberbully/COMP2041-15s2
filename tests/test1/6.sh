#!/bin/sh

# Enter pipeline/shell script below.

sort -n -t "," -k 3 marks.txt | cut -d "," -f 3 | uniq -u | head -7
