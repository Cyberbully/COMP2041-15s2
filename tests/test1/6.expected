8
11
13
18
20
21
27
